backend/development.ini.sample

```diff
@@ -586,7 +586,7 @@ website.usage_conditions =
 ## - 'individual': send a mail for every notification received
 ## - 'none': doesn't send any mail
 ## NB: new users will not being notified until they login to Tracim a first time
-; email.notification.type_on_invitation = summary
+; email.notification.type_on_invitation = daily
```

Comment:
Is the migration of existing instance (with their config file) has been taken into account ?

If I read correctly, the `summary` value no longer exist. To be backwards compatible, I suggest to continue accept `summary` value, but to print a deprecated warning and convert it into `daily`.

Reply:
I not understand this.

```
## send some notification when user make specific action in Tracim
; email.notification.activated = False
## you can change the type of email notification when a user is invited to a space by default.
## possible values are:
## - 'summary': send a daily mail containing notification summary
## - 'individual': send a mail for every notification received
## - 'none': doesn't send any mail
## NB: new users will not being notified until they login to Tracim a first time
; email.notification.type_on_invitation = summary
# You can enable notification-specific logs using the 'tracim_email_notification' logger.
```

summary exist already for daily notification. Why change this naming ? (breaking change ?) If it is necessary, change also description of value available.

Reply:
i have question also about naming because we have change naming in last year and is it now a good naming for the feature ?

Reply:
@buxx I agree for the background compatibility, that should not be complicated to implement.

@PhilippeAccorsi The naming change because until 4.8.0, the 3 options were:
- None
- Individual
- Summary
And summary meant "you get a mail containing the summary of your notifications"

Now, the options are:
- None
- Individual
- Hourly
- Daily
- Weekly
The 3 options hourly, daily and weekly are a "summary" option and we need to be able to differentiate them.

The documentation on the configuration in development.ini.sample hasn't been updated indeed, I'll fix that

Reply:
doc updated